llvm/test/Transforms/SCCP/pointer-nonnull.ll

```diff
@@ -232,9 +232,13 @@ define i1 @ip_test_nonnull_caller(ptr %p) {
 }
 
 define ptr @ret_nonnull_pointer(ptr nonnull %p) {
-; CHECK-LABEL: define ptr @ret_nonnull_pointer(
-; CHECK-SAME: ptr nonnull [[P:%.*]]) {
-; CHECK-NEXT:    ret ptr [[P]]
+; SCCP-LABEL: define ptr @ret_nonnull_pointer(
```

Comment:
Currently we only infer attributes in IPSCCP. I don't think there's any reason to not do it in SCCP as well, so that's another followup improvement...